packages/strapi/lib/middlewares/gzip/index.js

```diff
@@ -12,7 +12,8 @@ module.exports = strapi => {
      */
 
     initialize() {
-      strapi.app.use(compress());
+      const { options = { br: false } } = strapi.config.middleware.settings.gzip;
```

Comment:
you can leave the default to be an empty object

Reply:
Do we want `br: true` when gzip `enabled: true` but has `no options`?
Because if there are `no options` then i think the `br` will be enabled [(see)](https://github.com/koajs/compress/blob/master/lib/index.js#L33)

As `br` is a new feature by default it should be disabled

Reply:
Indeed with the koa-compress upgrade it will add the brotli compression. I don't see any reason to disable it as it will not be a breaking change. this just adds support for it but.

I think just passing en empty options object by default would be enough

Reply:
Done